bin/prepare_environment.sh

```diff
@@ -61,22 +63,14 @@ echo
 read -p "Enter your email: " email
 echo # newline
 
-## Save username and password as environment variables
-
-echo "$username" > /tmp/user_data
-echo "$password" >> /tmp/user_data
-echo "$email" >> /tmp/user_data
-
-if ! npm list -g bcrypt; then
-    npm install -g bcrypt
-fi
-
-hashed_password=$(node -e "console.log(require('bcrypt').hashSync(process.argv[1], 10));" "$password")
-escaped_hashed_password=$(printf '%s\n' "$hashed_password" | sed -e 's/[\/&]/\\&/g')
+# Hash the password
+docker pull epicsoft/bcrypt
+encrypted_password=$(docker run --rm epicsoft/bcrypt hash "$password" 10 | sed -e 's/[\/&]/\\&/g')
+docker rmi epicsoft/bcrypt
 
 ## Replace the example_user and example_pass strings with the new user and password
 
```

Comment:
There is a bug with the position of setting.js, it should be fixed by removing `../ before the filename

Reply:
I'm referring to this part of the code, sorry
<img width="645" alt="Captura de pantalla 2024-07-09 a las 16 40 42" src="https://github.com/statuscompliance/infrastructure/assets/80347035/f6fd1441-537a-4456-971a-4f82f3bfd1bf">

Reply:
@alvarobernal2412 Good catch! Pushed the fix now!